### DISCUSSION TITLE
[ question ] When is the release planned for Windows

### DISCUSSION THREAD

**Discussion by root9464** [zed-industries/zed#16249] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hello, I apologize right away for using issue for other purposes, but I have a question for the development team. When is a full-fledged release for Windows planned? 

### If applicable, add mockups / screenshots to help present your vision of the feature

-

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/discussions/13911#discussioncomment-9978935

> Consider asking generic questions in https://github.com/zed-industries/zed/discussions next time.
> 
> Windows support is released when the time comes.
> 
> Currently, no work in that area is done or planned by Zed team, the whole development happens thanks to our great community contributors.
